helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -64,6 +64,14 @@
  *   },
  *   "expires" : null,
  *   "gracePeriod": 60,
+ *   "rolloutOptions": {
+ *     "migrate": false,
+ *     "parallelism": 2,
+ *     "timeout": 1000,
+ *     "overlap": true,
+ *     "token": "insecure-access-token",
+ *     "ignoreFailures": false
+ *   },
```

Comment:
I don't think `migrate` and `token` make sense to allow in a job.

Reply:
`token` is already in the job. `migrate` doesn't make sense, but it seems more work and potentially more confusing to exclude some RolloutOption fields than to include everything.

Reply:
Just to toss the idea out there, you could introduce a new type ("JobRolloutOptions", for instance) with just the relevant fields, and then have the merging code/function be capable of producing a RolloutOptions from this type and the existing values/defaults.